images/base/Dockerfile

```diff
@@ -141,6 +143,7 @@ RUN git clone --filter=tree:0 "${RUNC_CLONE_URL}" /runc \
     && cd /runc \
     && git checkout "${RUNC_VERSION}" \
     && eval "$(gimme "${GO_VERSION}")" \
+    && export GOTOOLCHAIN="go${GO_VERSION}" \
```

Comment:
line159 seems is missing this export too

Reply:
yes. I will add this and bump up the go versions again

Reply:
fixed this one.